### ISSUE TITLE
[Screen Reader -Makecode-Documentation] Screen reader is not reading proper information of 'Led screen' in Scan mode

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2806]
**User Experience:**
Users who depends on Screen reader will get confuse if Narrator does not read proper information of led screen in Scan Mode

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge

**Repro Steps**
1-Navigate to url:-https://makecode.microbit.org/reference/basic/show-number
2-Navigate to all element on show number page
3- launch narrator change to scan mode by (caps+ space key)
4-Verify if screen reader is reading led screen information correctly or not

**Actual Result**
In Scan mode(CAPS + Space+ Down/Arrow key), Screen reader is not reading proper information of led screen 'For loop'

**Expected Result:**
Scan mode(CAPS + Space+ Down/Arrow key), Screen reader must read proper information of led screen 'For loop' 

**Note:**
Same issue exists In Item mode(CAPS + UP/Down Arrow key)

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

**Refer Attachment:**
[MAS4.2.1_NotReadingInfo.zip](https://github.com/Microsoft/pxt/files/1248874/MAS4.2.1_NotReadingInfo.zip)

**Comment by veler** [comment]
This issue is part of Blockly which is a third part component.

**Comment by abchatra** [comment]
We have got exception for Blockly bugs from Julia Liuson (CVP). Closing these as Won't Fix. Following is the reasoning.
----------------------
MakeCode can be used in either Text editor or Block editor.

Blocks editor is inherently visual and is based on google blockly. Usage requires drag and drop functionality and we don’t have a equivalence in MakeCode for this w.r.t accessibility. 
We have evaluated couple of research projects which tries to find a equivalence to block based coding for accessibility needs: 
•	Project Torino (Requires students to buy physical blocks)
•	Accessible blockly (This is text based coding without syntax errors)
Both these are drastic departure from the blockly and provides a complete new experience. These are still in research and we intend not to use them in MakeCode at this point of time.

As workaround, the text editor in MakeCode is the way we see students with the need of accessibility using MakeCode.
